test/functional/test_framework/test_framework.py

```diff
@@ -158,6 +161,21 @@ def main(self):
             self.options.tmpdir = tempfile.mkdtemp(prefix=TMPDIR_PREFIX)
         self._start_logging()
 
+        # Seed the PRNG.  Note that testruns are only reproducible if only a
```

Comment:
nit: git grepping shows "test run" is the usual spelling in this repo

Reply:
changed to 'test run'